### PR TITLE
HomeWork3.26.03

### PR DESCRIPTION
**Pull request by Java-Dev-Art**

### CODE REVIEW

File: src/HomeWork4/Classes/Author.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
+package HomeWork4.Classes;
+
+public class Author extends Book{
+    private String name;
+    private String lastName;
+    private int age;
+    private String country;
+    public void setAuthor(String name, String lastName, int age,String country){
+        this.name = name;
```
Comment (svetamolchanova) [inline src/HomeWork4/Classes/Author.java:10]: for what we use this method?

File: src/HomeWork4/Classes/Author.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+    public String getCountry() {
+        return country;
+    }
+
+    public String toString(){
+        return "Author \nName : " + getName() + "\nLast Name : " + getLastName() + "\nAge : " + getAge() + "\nCountry : " + getCountry()
+                + "\n" + bookToString() + "\n" +genreToString() + "\n**************************" ;
+
+
```
Comment (svetamolchanova) [inline src/HomeWork4/Classes/Author.java]: unnecessary spaces

File: src/HomeWork4/Classes/Main.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+        author.setFantasy();
+        Library library = new Library();
+        library.setLibrary("Jagellonian Uniwersity","Poland");
+        library.setNameBook("Road is my Life");
+        library.setAuthor("Artur","Markowski",30,"Belarus");
+
+        System.out.println(author);
+        System.out.println(library);
+
+
```
Comment (svetamolchanova) [inline src/HomeWork4/Classes/Main.java]: unnecessary spaces

File: src/HomeWork4/Classes/Author.java (hunk 1 above)
Comment (Java-Dev-Art) [reply src/HomeWork4/Classes/Author.java:10]: I want to set all information about Author and work with this information.

**Review by svetamolchanova — APPROVED** [review]

**Pull request merged**